src/cascadia/TerminalApp/TabManagement.cpp

```diff
@@ -891,19 +894,72 @@ namespace winrt::TerminalApp::implementation
     // Arguments:
     // - sender: the control that originated this event (TabViewItem)
     // - eventArgs: the event's constituent arguments
-    void TerminalPage::_OnTabClick(const IInspectable& sender, const Windows::UI::Xaml::Input::PointerRoutedEventArgs& eventArgs)
+    void TerminalPage::_OnPointerPressed(const IInspectable& sender, const Windows::UI::Xaml::Input::PointerRoutedEventArgs& eventArgs)
     {
         if (eventArgs.GetCurrentPoint(*this).Properties().IsMiddleButtonPressed())

```

Comment:
BTW shouldn't we pass `sender` instead of `*this` if anything? The documentation makes it sound like we can also just pass `nullptr` btw: https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.input.pointerroutedeventargs.getcurrentpoint

Reply:
This was preexisting code, so I didn't think about it, but you are right `nullptr` seems to be correct in our case.